spec/fixtures/education_benefits_claims/10203/minimal.spl

```diff
@@ -57,7 +57,8 @@ Program Name: Degree Name
 
 Provide the location where you plan to or will start training:
 School Name
-Test, TN
+Test,
```

Comment:
Don't know that it is a big thing but do you want the comma displaying if there is no state?

Reply:
Thanks for checking, yes, that's expected for input into TIMS